File: frontend/src/clientes-estadistica-app/src/app/interfaces/clienteConBalance.ts
import { ICliente } from './cliente';

export interface IClienteConBalance {
  
  nombre: string;
  edad: number;
  sexo: string;
  pais: string;
  balance: number;
  numeroGastos: number;
  numeroIngresos: number;

  cliente: ICliente;
}

File: backend-estadistica/src/BackendEstadistica/Models/ClienteConBalance.cs
namespace BackendEstadistica.Models
{
    public class ClienteConBalance
    {
        public int ClienteId { get; set; }
        public string Nombre { get; set; }
        public int Edad { get; set; }
        public string Sexo { get; set; }
        public string Pais { get; set; }
        public double Balance { get; set; }
        public int NumeroGastos { get; set; }
        public int NumeroIngresos { get; set; }
    }

}
